src/content/en/fundamentals/instant-and-offline/get-started/textcontent-3.md

```diff
@@ -82,75 +72,45 @@ Looking at the bigger picture, the entire script block from which that snippet c
 HTML and CSS can be minified in the same way, so that you can achieve improved load times for 
 both formatting- and behavior-related code.
 
-Many online and desktop minification tools are available; here are some examples.
-
-### Minifier
+Many online and desktop minification tools are available; here's an example.
 
-This online tool minifies JavaScript or CSS via copy and paste. Paste the code, choose the 
+[Minifier](http://www.minifier.org/)
+is an online tool minifies JavaScript or CSS via copy and paste. Paste the code, choose the 
 type, and click `Minify`.
 
-[http://www.minifier.org/](http://www.minifier.org/)
-
 ![Minifier](images/image_300.png)
 
-### HTML Minifier
-
-Here's an online tool that also handles HTML, and automatically identifies the code type(s).
-
-[http://www.willpeavy.com/minifier/](http://www.willpeavy.com/minifier/)
-
-![HTML Minifier](images/image_301.png)
-
-### Kangax HTML Minifier
+Other minification tools include:
 
+- [HTML Minifier](http://www.willpeavy.com/minifier/):
+This online tool also handles HTML, and automatically identifies the code type(s).
+- [Kangax HTML Minifier](https://kangax.github.io/html-minifier/):
 This online tool offers a broad array of output customization options for the minified code.
-
-[https://kangax.github.io/html-minifier/](https://kangax.github.io/html-minifier/)
-
-![Kangax HTML Minifier](images/image_302.png)
-
-### NPM Packages
-
-The NPMJS site also has various minification packages, for Grunt
-
-[https://www.npmjs.com/package/grunt-html-minify](https://www.npmjs.com/package/grunt-html-minify)
-
-and Gulp,
-
-[https://www.npmjs.com/package/gulp-html-minifier](https://www.npmjs.com/package/gulp-html-minifier)
-
-and a popular package 
-
-[https://www.npmjs.com/package/html-minifier](https://www.npmjs.com/package/html-minifier)
-
-that includes a useful chart comparing its compression results with other methods.
+- [NPM for Grunt](https://www.npmjs.com/package/grunt-html-minify):
+An NPM minification package that integrates into the Grunt workflow.
+- [NPM for Gulp](https://www.npmjs.com/package/gulp-html-minifier):
+An NPM minification package that integrates into the Gulp workflow. 
+- [NPM HTML Minifier](https://www.npmjs.com/package/html-minifier):
+An NPM package that includes a useful chart comparing its compression results with other methods.
 
```

Comment:
The nomenclature here should probably be "Node module for Grunt", "Node module for Gulp" etc